main/acle.md

```diff
@@ -1509,24 +1509,32 @@ SVE language extensions:
 > [The __ARM_FEATURE_SVE_BITS macro](#the-__arm_feature_sve_bits-macro)
 > for details.
 
-**`__ARM_FEATURE_SVE_VECTOR_OPERATORS==1`**
+**`__ARM_FEATURE_SVE_VECTOR_OPERATORS==N`**
 
-> This indicates that applying the `arm_sve_vector_bits` attribute
+> When N is 1, this indicates that applying the `arm_sve_vector_bits` attribute
 > to an SVE vector type creates a type that supports the GNU vector
-> extensions. The state of this macro is only meaningful when
+> extensions. This value of this macro is only meaningful when
 > `__ARM_FEATURE_SVE_BITS` is nonzero. See [`arm_sve_vector_bits` behavior
 > specific to vectors](#arm_sve_vector_bits-behavior-specific-to-vectors)
 > for details.
 
+> When N is 2, this indicates that the operators outlined in the GNU vector
+> extensions additionally work on all SVE vector types. This value of this
+> macro still has meaning when `__ARM_FEATURE_SVE_BITS` is nonzero.
+
 **`__ARM_FEATURE_SVE_PREDICATE_OPERATORS==1`**
 
-> This indicates that applying the `arm_sve_vector_bits` attribute to
+> When N is 1, this indicates that applying the `arm_sve_vector_bits` attribute to
 > `svbool_t` creates a type that supports basic built-in vector operations.
 > The state of this macro is only meaningful when `__ARM_FEATURE_SVE_BITS`
 > is nonzero. See [`arm_sve_vector_bits` behavior specific to
 > predicates](#arm_sve_vector_bits-behavior-specific-to-predicates)
 > for details.
 
+> When N is 2, this indicates that the operators outlined in the GNU vector
```

Comment:
The GNU vector extensions don't currently support packed boolean vectors.  The section linked above tries to make a conservative stab at what such an extension might support in future.

Now that Clang does have generic support for packed boolean vectors, it might be worth changing the documentation for predicates to refer to the Clang extension instead of the GNU one.